[2025-03-23 22:42:07,682] INFO 172.22.0.1 - - [23/Mar/2025 22:42:07] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=/usr/local/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-03-23 22:42:07,683] INFO 172.22.0.1 - - [23/Mar/2025 22:42:07] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=/usr/local/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-03-23 22:42:07,796] INFO 172.22.0.1 - - [23/Mar/2025 22:42:07] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=/usr/local/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-03-23 22:42:07,799] INFO 172.22.0.1 - - [23/Mar/2025 22:42:07] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=/usr/local/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-03-23 22:42:17,090] INFO Adding saving goal with name: 'Pretty dress' - call_trace=/app/app.py L39  
[2025-03-23 22:42:17,099] INFO Saving goal added successfully - call_trace=/app/app.py L52  
[2025-03-23 22:42:17,102] INFO 172.22.0.1 - - [23/Mar/2025 22:42:17] "POST /goals HTTP/1.1" 200 - - call_trace=/usr/local/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-03-23 22:42:28,402] INFO Retrieving saving goal with ID: 1 - call_trace=/app/app.py L107 
[2025-03-23 22:42:28,404] INFO Saving goal with ID 1 found - call_trace=/app/app.py L120 
[2025-03-23 22:42:28,404] INFO 172.22.0.1 - - [23/Mar/2025 22:42:28] "GET /goals/{goal_id}?goal_id=1 HTTP/1.1" 200 - - call_trace=/usr/local/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-03-23 22:42:41,104] INFO  * Detected change in '/app/schemas/saving_goal.py', reloading - call_trace=/usr/local/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-03-23 22:42:43,041] INFO Retrieving saving goal with ID: 1 - call_trace=/app/app.py L107 
[2025-03-23 22:42:43,046] INFO Saving goal with ID 1 found - call_trace=/app/app.py L120 
[2025-03-23 22:42:43,047] INFO 172.22.0.1 - - [23/Mar/2025 22:42:43] "GET /goals/{goal_id}?goal_id=1 HTTP/1.1" 200 - - call_trace=/usr/local/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-03-23 22:42:46,354] INFO Retrieving saving goal with ID: 1 - call_trace=/app/app.py L107 
[2025-03-23 22:42:46,356] INFO Saving goal with ID 1 found - call_trace=/app/app.py L120 
[2025-03-23 22:42:46,357] INFO 172.22.0.1 - - [23/Mar/2025 22:42:46] "GET /goals/{goal_id}?goal_id=1 HTTP/1.1" 200 - - call_trace=/usr/local/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-03-23 22:42:47,067] INFO Retrieving saving goal with ID: 1 - call_trace=/app/app.py L107 
[2025-03-23 22:42:47,068] INFO Saving goal with ID 1 found - call_trace=/app/app.py L120 
[2025-03-23 22:42:47,069] INFO 172.22.0.1 - - [23/Mar/2025 22:42:47] "GET /goals/{goal_id}?goal_id=1 HTTP/1.1" 200 - - call_trace=/usr/local/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-03-23 22:42:58,627] INFO Retrieving saving goal with ID: 1 - call_trace=/app/app.py L107 
[2025-03-23 22:42:58,629] INFO Saving goal with ID 1 found - call_trace=/app/app.py L120 
[2025-03-23 22:42:58,629] INFO 172.22.0.1 - - [23/Mar/2025 22:42:58] "GET /goals/{goal_id}?goal_id=1 HTTP/1.1" 200 - - call_trace=/usr/local/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-03-23 22:42:59,428] INFO Retrieving saving goal with ID: 1 - call_trace=/app/app.py L107 
[2025-03-23 22:42:59,430] INFO Saving goal with ID 1 found - call_trace=/app/app.py L120 
[2025-03-23 22:42:59,431] INFO 172.22.0.1 - - [23/Mar/2025 22:42:59] "GET /goals/{goal_id}?goal_id=1 HTTP/1.1" 200 - - call_trace=/usr/local/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-03-23 22:44:43,546] INFO  * Detected change in '/app/schemas/saving_goal.py', reloading - call_trace=/usr/local/lib/python3.12/site-packages/werkzeug/_internal.py L97  
